data/data/bootstrap/baremetal/files/etc/NetworkManager/dispatcher.d/30-local-dns-prepender

```diff
@@ -6,13 +6,26 @@ case "$STATUS" in
     logger -s "NM local-dns-prepender triggered by ${1} ${2}."
     DNS_IP="127.0.0.1"
     set +e
-    logger -s "NM local-dns-prepender: Checking if local DNS IP is the first entry in resolv.conf"
-    if grep nameserver /etc/resolv.conf | head -n 1 | grep -q "$DNS_IP" ; then
-        logger -s "NM local-dns-prepender: local DNS IP already is the first entry in resolv.conf"
-        exit 0
+    if systemctl -q is-enabled systemd-resolved; then
```

Comment:
Ahh whee the resolved thing.  I have only been tangentially following this, I think @dustymabe took the brunt of it.

This duplicates code that landed in the MCD right?  I think it's going to be a long term win if we change the bootstrap process to extract config files from the MCD instead of duplicating things in the installer.

Reply:
Right. I'd love to reuse the code, but prepender runs before FCOS pivots to OKD payload.

What we could do is to have a separate set of config, fetched from the image in the payload perhaps?